log-exploration-api-namespace.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+  name: openshift-logging
+  labels:
+    openshift.io/cluster-monitoring: "true" # enables this namespace to be monitored by openshift
```

Comment:
It would be nice to add a section to `README.md` about how to enable metrics from `log-exploration-api`

Reply:
Updated README.md with notes on how to enable metrics from log-exploration-api.